cmd/installer/root.go

```diff
@@ -420,6 +420,50 @@ func UpdateInstalledPDSCFiles(pidxXML *xml.PidxXML, concurrency int, timeout int
 		}
 	}
 
+	pdscFiles, err = utils.ListDir(Installation.LocalDir, ".pdsc$")
+	if err != nil {
+		return err
+	}
+
+	numPdsc = len(pdscFiles)
+	if utils.GetEncodedProgress() {
+		log.Infof("[J%d:F\"%s\"]", numPdsc, Installation.PublicIndex)
+	}
+
+	for _, pdscFile := range pdscFiles {
+		log.Debugf("Checking if \"%s\" needs updating", pdscFile)
```

Comment:
More precisely you are checking whether you can find and read a pdsc file to update the copy in .Local 
Needs updating sounds as if you can actually check whether there is a newer version available.

Reply:
I check at the end, if the copied pdsc file contains another highest version than the old one in .local. Nearly the same as in line 371 with the public pdsc files.